website/server/controllers/api-v3/challenges.js

```diff
@@ -659,7 +659,7 @@ api.exportChallengeCsv = {
 };
 
 /**
- * @api {put} /api/v3/challenges/:challengeId Update the name, description, or leader of a challenge
+ * @api {put} /api/v3/challenges/:challengeId Update the name or description of a challenge
```

Comment:
since it looks like the summary can be changed as well maybe this should say `Update the name, description, or summer of a challenge	 `

Reply:
I made the change